tests/conftest.py

```
@@ -17,27 +23,20 @@
-from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession, async_sessionmaker
-
 # --- App and Model Imports ---
-from app.main import app
-from app.core.base_model import Base
-from app.core.database import get_db, get_async_db
-from app.modules.auth.auth_service import AuthService
-from app.modules.usage.usage_service import UsageService
-from app.modules.users.user_model import User
-from app.modules.projects.projects_model import Project
-from app.modules.conversations.conversation.conversation_model import (
+from app.main import app  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)